conf/airframes/TUDELFT/tudelft_conf.xml

```diff
@@ -430,6 +430,28 @@
    gui_color="green"
   />
   <aircraft
+   name="ardrone2_flightplan_guided_gazebo"
```

Comment:
Why add simulation specific vehicles? The idea of the targets is that one airframe can be used for both real flight and simulation.

Reply:
Good point. Only: the orangeavoider runs in a bebop and gazebo does not simulate that yet.

Reply:
There are now 3 types of vehicles defined. ARdrone, simple_quad and simple_x_quad. If you would like a bebop specific vehicle you can add it and define the type in the airframe using NPS_GAZEBO_AC_NAME. I think it would be sufficient however just to use the default simple_x_quad  though.

If you are referring to the resolution of the camera, the you would indeed need a new vehicle. Or wait till the end of the week when I implement a way to dynamically add sensors to the models and then add the bebop camera instead of the ardrone camera to the default drone.

One major thing that is not yet implemented that may affect the bebop integration is there are no actuator dynamics implemented to INDI doesn't work. What I have done so far is use INDI in ap and quat_int in nps. I think that this is a fine compromise for now though.

Reply:
https://github.com/paparazzi/paparazzi/blob/master/conf/airframes/KS/ks_bebop2_stereo.xml